### PR TITLE
added compose example with JMX

### PR DESCRIPTION
**Pull request by coughman**

### CODE REVIEW

**Comment by ewencp** [comment]
@coughman who should review? core team? clients? can you assign one of the teams as reviewer?

**Comment by coughman** [comment]
assigned @aayars and clients

**Review by aayars — COMMENTED** [review]
Don't we need to set offset topic replication to 1 here?

**Comment by edenhill** [comment]
@mageshn Can you review this please?

**Comment by coughman** [comment]
@joel-hamill can you proof read the README file?

**Comment by coughman** [comment]
@aayars fixed replication factor in latest commit

File: examples/JMX/docker-compose.yaml
Comment (mageshn) [inline examples/JMX/docker-compose.yaml]: why KAFKA_REST in zk service

File: examples/JMX/docker-compose.yaml
Comment (mageshn) [inline examples/JMX/docker-compose.yaml]: similar to above, kafka_rest in kafka doesn't seem right

File: examples/JMX/docker-compose.yaml
Comment (mageshn) [inline examples/JMX/docker-compose.yaml]: shouldn't KAFKAREST_JMX_OPTS be KAFKA_REST_JMX_OPTS

**Review by mageshn — CHANGES_REQUESTED** [review]

File: examples/JMX/docker-compose.yaml
Comment (coughman) [reply examples/JMX/docker-compose.yaml]: fixed in latest commit

**Comment by coughman** [comment]
I removed all unrelated envs from all services

File: examples/JMX/README.md
Comment (joel-hamill) [inline examples/JMX/README.md]: suggest `...all Kafka services use the..`

Comment (joel-hamill) [inline examples/JMX/README.md]: suggest: `This compose example is configured...`

File: examples/JMX/README.md
Comment (joel-hamill) [inline examples/JMX/README.md]: suggest: `If you are running Kafka CLI commands locally within the container, you must unset the `*JMX_OPTS` environment variable.`

**Review by joel-hamill — APPROVED** [review]
grammar fixes

**Comment by coughman** [comment]
incorporated changes suggested by @joel-hamill

File: examples/JMX/docker-compose.yaml
Comment (coughman) [reply examples/JMX/docker-compose.yaml]: removed KAFKA_REST* env

File: examples/JMX/docker-compose.yaml
Comment (coughman) [reply examples/JMX/docker-compose.yaml]: unfortunately KAFKAREST_JMX_OPTS is used in kafka-rest repo:
https://github.com/confluentinc/kafka-rest/blob/1cb07d62871d8eb24cfeb5939bbe4fc5aa308037/bin/kafka-rest-run-class#L43-L45

File: examples/JMX/docker-compose.yaml
Comment (lostick) [inline examples/JMX/docker-compose.yaml]: Shouldn't it be `KAFKA_REST_JMX_OPTS` ?

**Pull request closed without merging**
